pyproject.toml

```diff
@@ -100,6 +100,10 @@ force-exclude = '''
 ^/test-data
 '''
 
+[tool.codespell]
+ignore-words-list = "ans,ccompiler,corus,fo,froms,haa,hax,notin,ot,statics,whet,zar"
+skip = "./mypyc/external/*,*.pyi,*.test"
```

Comment:
I do wonder if we should just skip the whole `mypy/test/` and `mypyc/test` folders entirely.

Personally I'd also just move the config to `.pre-commit-config.yaml` directly. Don't think any one is ever running codespell outside pre-commit.

This also allows us to use the pre-commit files filter as well with `exclude_types: [pyi, test]` and `exclude: ^(mypyc/external/|mypy/test|mypyc/test)`.